applications/GeoMechanicsApplication/custom_utilities/math_utilities.hpp

```diff
@@ -78,55 +75,58 @@ class GeoMechanicsMathUtilities
+        solution.resize(3, false);
+        noalias(solution) = ZeroVector(3);
 
-        if (a==0)
-        {
-            std::cout<<"This is not a cubic equation: CardanoFormula"<<std::endl;
+        if (a == 0) {
+            std::cout << "This is not a cubic equation: CardanoFormula" << std::endl;
 
             return false;
         }
 
-        double p= (3.0*a*c-b*b)/(3.0*a*a);
+        double p = (3.0 * a * c - b * b) / (3.0 * a * a);
 
-        double q= 2.0*b*b*b/(27.0*a*a*a)-b*c/(3.0*a*a)+d/a;
+        double q = 2.0 * b * b * b / (27.0 * a * a * a) - b * c / (3.0 * a * a) + d / a;
 
-        double discriminante= p*p*p/27.0+q*q/4.0;
+        double discriminante = p * p * p / 27.0 + q * q / 4.0;
 
-        if(discriminante>0)
-        {
+        if (discriminante > 0) {
             return false;
         }
 
-        if(discriminante==0)
-        {
-            if( a == 0 )
-                return false;
+        if (discriminante == 0) {
+            if (a == 0) return false;
 
-            solution(0)= pow(q/2.0, 1.0/3.0)-b/(3*a);
-            solution(1)= pow(q/2.0, 1.0/3.0)-b/(3*a);
-            solution(2)= pow(-4.0*q, 1.0/3.0)-b/(3*a);
+            solution(0) = pow(q / 2.0, 1.0 / 3.0) - b / (3 * a);
+            solution(1) = pow(q / 2.0, 1.0 / 3.0) - b / (3 * a);
+            solution(2) = pow(-4.0 * q, 1.0 / 3.0) - b / (3 * a);
 
             return true;
         }
 
-        if(p < 0) return false; //in this case the square roots below will be negative. This substitutes with better efficiency lines 107-110
-        
-        solution(0) = -sqrt(-4.0/3.0*p)*cos(1.0/3.0*acos(-q/2.0*sqrt(-27.0/(p*p*p)))+ Globals::Pi / 3.0) -b/(3*a);
-        solution(1) =  sqrt(-4.0/3.0*p)*cos(1.0/3.0*acos(-q/2.0*sqrt(-27.0/(p*p*p))))-b/(3*a);
-        solution(2) = -sqrt(-4.0/3.0*p)*cos(1.0/3.0*acos(-q/2.0*sqrt(-27.0/(p*p*p)))- Globals::Pi / 3.0) -b/(3*a);
+        if (p < 0)
+            return false; // in this case the square roots below will be negative. This substitutes with better efficiency lines 107-110
+
+        solution(0) = -sqrt(-4.0 / 3.0 * p) *
+                          cos(1.0 / 3.0 * acos(-q / 2.0 * sqrt(-27.0 / (p * p * p))) + Globals::Pi / 3.0) -
+                      b / (3 * a);
+        solution(1) = sqrt(-4.0 / 3.0 * p) * cos(1.0 / 3.0 * acos(-q / 2.0 * sqrt(-27.0 / (p * p * p)))) -
+                      b / (3 * a);
+        solution(2) = -sqrt(-4.0 / 3.0 * p) *
+                          cos(1.0 / 3.0 * acos(-q / 2.0 * sqrt(-27.0 / (p * p * p))) - Globals::Pi / 3.0) -
+                      b / (3 * a);
 
-//        if(std::isnan<double>(solution(0)) || std::isnan<double>(solution(1))|| std::isnan<double>(solution(2)))
-//        {
-//            return false;
-//        }
+        //        if(std::isnan<double>(solution(0)) || std::isnan<double>(solution(1))|| std::isnan<double>(solution(2)))
+        //        {
+        //            return false;
+        //        }
```

Comment:
Perhaps, is it good to remove the commented lines?

Reply:
Yes, but in my opinion it should be done in a separate commit to master (such that we only do formatting here). I'll make a separate PR to remove commented code in custom_utilities.